recipes/pyside/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+if test `uname` = "Darwin"
+then
+  SO_EXT='.dylib'
+else
+  SO_EXT='.so'
+fi
+
+
+pushd sources/shiboken
+mkdir -p build && cd build
+
+export QTDIR=${PREFIX}
+
+PY_LIB=`find $PREFIX/lib -name libpython${PY_VER}*${SO_EXT}`
+PY_INC=`find $PREFIX/include -name python${PY_VER}*`
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DCMAKE_INSTALL_RPATH=${PREFIX}/lib \
+  -DBUILD_TESTS=OFF \
+  -DUSE_PYTHON3=${PY3K} \
+  -DPYTHON3_INCLUDE_DIR=${PY_INC} \
+  -DPYTHON3_LIBRARY=${PY_LIB} \
+  ..
+make install -j${CPU_COUNT}
+
+
+popd
+pushd sources/pyside
+
+mkdir -p build && cd build
+
+cmake \
+  -DCMAKE_FIND_ROOT_PATH=${PREFIX} \
+  -DCMAKE_INSTALL_PREFIX=${PREFIX} \
+  -DCMAKE_BUILD_TYPE=Release \
+  -DUSE_XVFB=ON \
+  ..
+make install -j${CPU_COUNT}
+DYLD_FALLBACK_LIBRARY_PATH=${PREFIX}/lib ctest -j${CPU_COUNT} --output-on-failure  --timeout 100
```

Comment:
May need to do something like [this]( https://github.com/conda-forge/qtconsole-feedstock/blob/421859b54e7ead2c2193971aefda86c8e55dbc57/recipe/meta.yaml#L38 ) for these to work on Linux. So please add a branch to handle the two OS cases.